results-processor/wptscreenshot.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2019 The WPT Dashboard Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import base64
+import binascii
+import gzip
+import logging
+import multiprocessing
+import os
+import time
+
+import requests
+
+import config
+
+DATA_URI_PNG_PREFIX = 'data:image/png;base64,'
+
+_log = logging.getLogger(__name__)
+
+
+############################
+# Start of worker functions
+# These functions run in worker processes. DO NOT use _log.
+
+# Global variables to be initialized in workers:
+_api = 'API URL to be initialized'
+_run_info = {}
+
+
+def _initialize(api, run_info):
+    global _api
+    global _run_info
+    _api = api
+    _run_info = run_info
+
+
+def _upload(images):
+    files = []
+    for i in range(len(images)):
+        files.append((
+            'screenshot', (('{}.png').format(i), images[i], 'image/png')))
```

Comment:
Is `.format` preferred to `'%s.png' % i`?

Reply:
Not really. They have subtle differences which don't matter in this case. Let me change it to `%` for consistency.